mmv1/third_party/terraform/tests/resource_sql_user_test.go

```diff
@@ -262,6 +262,10 @@ resource "google_sql_user" "user1" {
   instance = google_sql_database_instance.instance.name
   host     = "google.com"
   password = "%s"
+	sql_server_user_details {
```

Comment:
nit: Can you indent these with spaces, similar to the rest of the lines in config?

Reply:
fixed